test/integration/model.test.js

```diff
@@ -1333,22 +1333,17 @@ describe(Support.getTestDialectTeaser('Model'), function() {
       return ParanoidUser.sync({ force: true }).then(function() {
         return ParanoidUser.bulkCreate(data);
       }).bind({}).then(function() {
-        // since we save in UTC, let's format to UTC time
-        this.date = moment().utc().format('YYYY-MM-DD h:mm');
         return ParanoidUser.destroy({where: {secretValue: '42'}});
       }).then(function() {
         return ParanoidUser.findAll({order: ['id']});
       }).then(function(users) {
         expect(users.length).to.equal(1);
         expect(users[0].username).to.equal('Bob');
 
-        return self.sequelize.query('SELECT * FROM ' + qi('ParanoidUsers') + ' WHERE ' + qi('deletedAt') + ' IS NOT NULL ORDER BY ' + qi('id'));
+        return self.sequelize.query('SELECT * FROM ' + qi('ParanoidUsers') + ' WHERE ' + qi('deletedAt') + ' IS NOT NULL AND ' + qi('deletedAt') + ' <= CURRENT_TIMESTAMP ORDER BY ' + qi('id'));
       }).spread(function(users) {
         expect(users[0].username).to.equal('Peter');
         expect(users[1].username).to.equal('Paul');
-
-        expect(moment(new Date(users[0].deletedAt)).utc().format('YYYY-MM-DD h:mm')).to.equal(this.date);
```

Comment:
I'm not sure why, but when I run this test the locally instantiated date is set to `1970-01-01 12:00` which is nowhere near the `CURRENT_TIMESTAMP` of the database, thereby causing the test to fail. Perhaps this is part of the test suite?

Regardless, this seems to highlight the problems that can arise when database and system clocks for whatever reason aren't synced. We *could* still compare the `deletedAt` field assigned by the database with the locally instantiated one, but seeing as the server time is no longer used to populate the database I'm not sure it buys us anything.